doc/api/fs.md

```diff
@@ -1524,11 +1524,27 @@ If `options` is a string, then it specifies the encoding. Example:
 ```js
 fs.readFile('/etc/passwd', 'utf8', callback);
 ```
+*Note*: When the path is a directory, the behavior of
+`fs.readFile()` and [`fs.readFileSync()`][] is platform-specific. On macOS, Linux
+and Windows, an error will be returned. On FreeBSD, the contents of the files
```

Comment:
@refack 
Did not see that! Got it. Fixed! 

Question:  ```[`fs.readFileSync()`][]``` 
Is this the right way to link this to the title `fs.readFileSync()` ?  
Doesn't seem to be working.